packages/components/src/components/wallet-icon/wallet-icon.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { CSSProperties } from 'react';
+import Icon from '../icon';
+import { GradientBackgroundWalletIcon } from '../gradient-background-wallet-icon';
+import './wallet-icon.scss';
+import classNames from 'classnames';
+
+type TWalletIcon = {
+    colors: {
+        primary: CSSProperties['backgroundColor'];
+        secondary: CSSProperties['backgroundColor'];
+    };
+    icon: string;
+    rounded?: string;
+    size: 'small' | 'medium' | 'large';
+};
+
+const WalletIcon = ({ colors, icon, rounded, size }: TWalletIcon) => {
```

Comment:
@aum-deriv This component seems redundant 🤔 I can just use the `TGradientBackgroundWalletIcon `component and give it a child 🤔 
```tsx
<GradientBackgroundWalletIcon
    size='small'
    color='#FFFFFF'
    primary='#0DB43D'
    secondary='#FFCD00'
>
    <Icon icon='IcCurrencyAud' size={16} />
</GradientBackgroundWalletIcon>
```
```tsx
<GradientBackgroundWalletIcon
    size='medium'
    color='#FFFFFF'
    primary='#0DB43D'
    secondary='#FFCD00'
>
    <Icon icon='IcCurrencyAud' size={24} />
</GradientBackgroundWalletIcon>
```
```tsx
<GradientBackgroundWalletIcon
    size='large'
    color='#FFFFFF'
    primary='#0DB43D'
    secondary='#FFCD00'
>
    <Icon icon='IcCurrencyAud' size={48} />
</GradientBackgroundWalletIcon>
```